wordpress-mobile/WordPress-iOS
# Dynamic Dashboard Card Image UI Fix

Fixes #22501 

## Testing Steps
1. Install & launch Jetpack App.
2. ✅  Verify that the "New: Domains Management" card UI does not jump around.
3. Switch to another tab, and open My Site again.
4. ✅  Verify that the "New: Domains Management" card UI looks correct.

## Screen Recordings

### Before

https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/7cb850b2-5cc1-4479-b1d9-1945f5c4e7a8

### After

https://github.com/wordpress-mobile/WordPress-iOS/assets/15968946/c93d3b62-58bb-4154-b064-0fd2d07b9881

## Regression Notes
1. Potential unintended areas of impact
N/A

6. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

7. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


UI Changes testing checklist:
